server/controllers/settings.js

```diff
@@ -35,12 +35,14 @@ export default {
     }
 
     const location = await locateAddress(req.body.address)
+    const receiptFields = req.body.receiptFields
 
     if (!location) return next(new ValidationError({address: 'Address not found'}))
 
     const settings = {
       ...req.body,
-      location
+      location,
+      receiptFields
```

Comment:
The `...req.body` above will also copy this, you shouldn't need either of the changes in this file.